### DISCUSSION TITLE
Issue with uniting metal and ligand generation metal-lignad

### DISCUSSION THREAD

**Discussion by CHIHX12** [rdkit/rdkit#4730] [Q&A]
![image](https://user-images.githubusercontent.com/51500438/142155068-ef551189-e6c1-4942-beca-fb77f97d9db4.png)

**Comment by greglandrum** [comment]
@zxc1bnm2 : there's not enough information here to understand what you are looking for as an enhancement.

**Comment by CHIHX12** [comment]
![image](https://user-images.githubusercontent.com/51500438/142195327-00090175-1558-4fb3-be4b-dc4813dd1254.png)

**Comment by greglandrum** [comment]
again. I can't figure out what you want to do here. Please provide a code example that demonstrates what you want.

**Comment by CHIHX12** [comment]
![image](https://user-images.githubusercontent.com/51500438/142836545-4a50c217-d204-4989-bf34-2cf063c68fba.png)

**Comment by greglandrum** [comment]
*[Meta]* I moved this into `Discussions` because I am trying to stop using the Issue tracker for questions and discussions.
